ocaml/xapi-idl/network/network_interface.ml

```diff
@@ -336,7 +336,12 @@ module Interface_API (R : RPC) = struct
 
   let clear_state =
     declare "clear_state"
-      ["Clear configuration state"]
+      ["Clear configuration state and pause writing to disk"]
+      (unit_p @-> returning unit_p err)
+
+  let sync_state =
```

Comment:
Can you change the name to `persist_state`? Sync can be quite a nebulous action, while persist is much more specific.